src/py/aspen/config/config.py

```diff
@@ -172,21 +174,40 @@ def AUTH0_DOMAIN(self) -> str:
 
     @property
     def AUTH0_BASE_URL(self) -> str:
-        return f"https://{self.AUTH0_DOMAIN}"
+        try:
+            return self.AWS_SECRET["AUTH0_BASE_URL"]
+        except KeyError:
+            return f"https://{self.AUTH0_DOMAIN}"
 
     @property
     def AUTH0_ACCESS_TOKEN_URL(self) -> str:
-        return f"{self.AUTH0_BASE_URL}/oauth/token"
+        try:
+            return self.AWS_SECRET["AUTH0_ACCESS_TOKEN_URL"]
+        except KeyError:
+            return f"{self.AUTH0_BASE_URL}/oauth/token"
 
     @property
     def AUTH0_AUTHORIZE_URL(self) -> str:
-        return f"{self.AUTH0_BASE_URL}/authorize"
+        try:
+            return self.AWS_SECRET["AUTH0_AUTHORIZE_URL"]
+        except KeyError:
+            return f"{self.AUTH0_BASE_URL}/authorize"
 
     @property
     def AUTH0_CLIENT_KWARGS(self) -> Mapping[str, Any]:
-        return {
-            "scope": "openid profile email",
-        }
+        try:
+            return self.AWS_SECRET["AUTH0_CLIENT_KWARGS"]
+        except KeyError:
+            return {
+                "scope": "openid profile email",
+            }
+
+    @property
+    def AUTH0_USERINFO_URL(self) -> str:
+        try:
+            return self.AWS_SECRET["AUTH0_USERINFO_URL"]
+        except KeyError:
+            return "userinfo"
```

Comment:
Why the fallbacks?  When do we expect the secret not to be present?  It seems you pre-populate the secretsmanager in the mock docker instance.

Reply:
Just for reverse-compatibility with the existing dev/staging environments.

Reply:
Where do we have environments that don't have secrets populated?

Reply:
@ttung the local dev environment uses oidc instead of auth0 so these secrets wouldn't be populated for local

Reply:
So couldn't the local dev environment define these functions?

Reply:
It's the current EBS staging env that doesn't.

Reply:
The current staging env doesn't define them because it falls back on the methods here, but the secret for the staging environment has these keys.  I'm asking which environment has a secret that doesn't have these keys defined.